setuptools/tests/test_ccompiler.py

```diff
@@ -0,0 +1,8 @@
+from setuptools.ccompiler import CCompiler
+from setuptools.ccompiler import customize_compiler
+from setuptools.ccompiler import new_compiler
+
+def test_ccompiler_namespace():
```

Comment:
```suggestion
def test_ccompiler_namespace(_avoid_permanent_changes_in_sysconfig):
```

(See the following suggestion so it makes sense)